README.md

```diff
@@ -96,51 +96,18 @@ Note, that in the example above, the subscription must be in the `marketplace` (
 
 After an operator has been installed, to uninstall the operator you need to delete the following resources. Below we uninstall the `jaeger` operator as an example.
 
-Delete the `Subscription` in the `targetNamespace` that the operator was installed into. If created via the OpenShift OperatorHub UI, it will be named after the operator's packageName.
+Delete the `Subscription` in the namespace that the operator was installed into. For upstream, this is the `marketplace` namespace, for OCP, this is the `openshift-marketplace` namespace. If the install was done via the OpenShift OperatorHub UI, it will be named after the operator's packageName so you can run the following command to delete it from the command line:
```

Comment:
>If the install was done via the OpenShift OperatorHub UI, it will be named after the operator's packageName so you can run the following command to delete it from the command line:

How will that command work in OpenShift given there is no `local-operators` namespace? And please call it OKD for consistency.